### PR TITLE
Update dotnet-framework.md installation note

### PR DESCRIPTION
**Pull request by jessicagamio**
Clarify the example shared in the Notes section of the installation guide regarding using only one apm solution in your application.
This is a duplicate of the following PR for DotNet core docs - [link to PR](https://github.com/DataDog/documentation/pull/23024)

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Comment by jessicagamio** [comment]
@zacharycmontoya and @urseberry please review

**Review by buraizu — APPROVED** [review]

**Comment by lucaspimentel** [comment]
This change should be done in the `dotnet-core.md` file as well.

Related: the "Please merge after reviewing" checkbox in the PR description was not checked. It would help to wait for the requested `tracing-dotnet` team review before merging.

**Pull request merged**
